tsconfig.json

```diff
@@ -14,7 +14,7 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "inlineSources": true,
-    "lib": [ "dom", "es6" ],
+    "lib": [ "dom", "es6", "es2017" ],
```

Comment:
do you think this will need to be added to the UI?

Reply:
It would be good to add it there as well but not necessary since build code from `ngx-login-client` will be used in `fabric8-ui`.

Reply:
FYI the monorepo will also include `es2017` as part of the common `tsconfig`.

Reply:
Yes, i did check that `tsconfig` in talamer has `es2017` along with `es2015` and `es2016`. :)